x-pack/plugins/ml/public/application/explorer/explorer_utils.ts

```diff
@@ -51,6 +54,13 @@ export interface ExplorerJob {
   selected: boolean;
   bucketSpanSeconds: number;
   isSingleMetricViewerJob?: boolean;
+  sourceIndices?: string[];
+}
+
+export function isExplorerJob(arg: any): arg is ExplorerJob {
```

Comment:
we should use `unknown` in type guards 
```suggestion
export function isExplorerJob(arg: unknown): arg is ExplorerJob {
```